pages/index.vue

```diff
@@ -1,5 +1,28 @@
-<!-- Remove these two lines once content is added -->
-<!-- eslint-disable-next-line vue/valid-template-root -->
-<template></template>
+<!-- eslint-disable vue/attribute-hyphenation -->
+<template>
+  <!-- long logo -->
+  <img class="mx-auto" src="../assets/logos/TransparentLongLogo.png" />
+  <!-- competitions -->
+  <h1 class="text-center font-montserrat text-4xl">Competitions</h1>
 
-<script setup></script>
+  <hr class="my-5 w-4/5 h-px mx-auto bg-black border-0" />
+
+  <div class="flex h-screen justify-center">
+    <div class="grid grid-cols-3 grid-rows-6 gap-6 justify-between">
+      <!-- replace key with item.key when final data exists -->
+      <div v-for="item in tournaments" :key="item" class="">
```

Comment:
```suggestion
      <div v-for="item in tournaments" :key="item">
```
The `class` attribute is empty.